CHANGELOG.md

```diff
@@ -8,121 +8,70 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [10.16.0]
 ### Uncategorized
```

Comment:
@rekmarks the Uncategorized PRs remaining are all the snaps, pending to curate and review from Snaps team

Reply:
TY @FrederikBolding  I've updated all the changes